ga4gh/datamodel/variants.py

```diff
@@ -585,10 +585,46 @@ def getMetadata(self):
         # All the metadata is available via each varFile.header, including:
         #    records: header records
         #    version: VCF version
-        #    samples
-        #    contigs
-        #    filters
+        #    samples -- not immediately needed
+        #    contigs -- not immediately needed
+        #    filters -- not immediately needed
         #    info
         #    formats
+
+        def buildMetadata( key, value, id, type, number, description ): # All input are strings
```

Comment:
It would be nice to have sensible defaults for these parameters here. How about
```python
def buildMetatdata(key, value="", id="", type="", number="", description=""):
     # etc
```